### DISCUSSION TITLE
PaperlessNGX add Tika and Gotenberg

### DISCUSSION THREAD

**Discussion by Citroene** [tteck/Proxmox#1808] [Ideas]
Hi,

Is it possible to add Tika and Gotenberg to the script?

> Paperless can make use of [Tika](https://tika.apache.org/) and [Gotenberg](https://gotenberg.dev/) for parsing and converting "Office" documents (such as ".doc", ".xlsx" and ".odt"). Tika and Gotenberg are also needed to allow parsing of E-Mails (.eml).

Kind regards.

**Comment by Stephan-4711** [comment]
woule be really great
